src/main/java/org/spongepowered/common/data/SpongeSerializationRegistry.java

```diff
@@ -406,12 +413,19 @@ public static void setupSerialization(Game game) {
         dataRegistry.registerDataProcessorAndImplBuilder(ExperienceHolderData.class, SpongeExperienceHolderData.class, ImmutableExperienceHolderData.class,
                                                          ImmutableSpongeExperienceHolderData.class, experienceHolderDataProcessor,
                                                          experienceHolderDataBuilder);
+
         final ItemAuthorDataProcessor itemAuthorDataProcessor = new ItemAuthorDataProcessor();
         final ItemAuthorDataBuilder itemAuthorDataBuilder = new ItemAuthorDataBuilder();
         service.registerBuilder(AuthorData.class, itemAuthorDataBuilder);
         dataRegistry.registerDataProcessorAndImplBuilder(AuthorData.class, SpongeAuthorData.class, ImmutableAuthorData.class,
                                                   ImmutableSpongeAuthorData.class, itemAuthorDataProcessor, itemAuthorDataBuilder);
 
+        final MovementSpeedDataProcessor movementSpeedDataProcessor = new MovementSpeedDataProcessor();
+        final MovementSpeedDataBuilder movementSpeedDataBuilder = new MovementSpeedDataBuilder();
+        service.registerBuilder(MovementSpeedData.class, movementSpeedDataBuilder);
+        dataRegistry.registerDataProcessorAndImpl(MovementSpeedData.class, SpongeMovementSpeedData.class, ImmutableMovementSpeedData.class,
+                ImmutableMovementSpeedData.class, movementSpeedDataProcessor, movementSpeedDataBuilder);
```

Comment:
Yes... because we want to register the `ImmutableMovementSpeedData.class` twice.